modules/apps/dynamic-data-mapping/dynamic-data-mapping-test/src/testIntegration/java/com/liferay/dynamic/data/mapping/upgrade/v5_1_1/test/DDMStructureVersionDDMStructureCTUpgradeProcessTest.java

```diff
@@ -26,7 +28,15 @@ public class DDMStructureVersionDDMStructureCTUpgradeProcessTest
 	protected CTModel<?> addCTModel() throws Exception {
 		DDMStructure ddmStructure = (DDMStructure)super.addCTModel();
 
-		return ddmStructure.getStructureVersion();
+		DDMStructureVersion ddmStructureVersion =
+			ddmStructure.getStructureVersion();
+
+		ddmStructureVersion.setDefinition(
+			DDMFormSerializeUtil.serialize(
+				getDDMForm(), _jsonDDMFormSerializer));
+
+		return _ddmStructureVersionLocalService.updateDDMStructureVersion(
+			ddmStructureVersion);
```

Comment:
I don't think we need this code.
Why?

Reply:
Hi @marcelabc,

This case is similar to the one before, those changes have in mind to make the class more generic as possible, I noticed I needed to have a bunch of override to the v3_7_1 tests when extending this class but as a matter of fact I didn't needed it, I could just rewrite some parts of the original code that it would make it work and also preserving the other existing tests without making them to fail. I tested the other tests as well both with and without the fix and the expected results happened. Therefore confirming that what I did made the class more generic and ready for more cases.